curp/tests/common/curp_group.rs

```diff
@@ -106,7 +108,7 @@ pub struct CrashedCurpNode {
 }
 
 pub struct CurpGroup {
-    pub nodes: HashMap<ServerId, CurpNode>,
+    pub nodes: HashMap<ServerId, (CurpNode, Arc<TestRoleChangeInner>)>,
```

Comment:
Would it be better to give it a named structure instead of a tuple?

Reply:
Done